Lib/test/test_dict_version.py

```diff
@@ -0,0 +1,191 @@
+"""
+Test implementation of the PEP 509: dictionary versioning.
+"""
+import unittest
+from test.support import import_helper
+
+# PEP 509 is implemented in CPython but other Python implementations
```

Comment:
I don't believe we implement it, unsure if this test file actually gets executed tbh.